### ISSUE TITLE
[QUESTION]

### ISSUE THREAD

**Issue by zilexa** [neffo/bing-wallpaper-gnome-extension#214]
Where did the pause button go ?
I do not understand why this was removed, perhaps it was a PLAY button. But it was the only one I used regularly: if there was a wallpaper I liked, I just hit pause. And then if I got tired of hit, hit the same button again to get the latest daily image. 

That's really all i expect from this extension. The ability to scroll through all images ever downloaded (does it clean up?) is also nice ofcourse. 

But I don't understand the heart button.. I'd just expect a play/pause button. Could that please be reintroduced?

I saw an option "Always show new images" under Settings. But that doesn't have the same definition as a simple pause. I do not want to change the actual behaviour, I might just want to pause it thats all. That option just seems confusion. Also, hidden.

**Comment by neffo** [comment]
Depending on which version you are using, it's now a toggle switch named 'always show new images' in quick settings:
![image](https://github.com/neffo/bing-wallpaper-gnome-extension/assets/5569112/da3c3f51-745a-4bcb-a77a-ddf1572d6aed)
Or it can also be found in the debug options. 

I've removed it as they weren't particularly obvious in their function, or that they were toggles.

**Comment by zilexa** [comment]
Yes that is what I thought. For me it was the most used button. Def not the dice.
This way it seems less intuitive to me. But at least i can still pause with this toggle.

**Comment by neffo** [comment]
We could reintroduce this as one of the icons on the action bar (with original behavior). The bar is a bit more crowded now, but will test it in the next version.

**Comment by zilexa** [comment]
Thank you!
